storm-core/src/jvm/storm/trident/state/map/MapCombinerAggStateUpdater.java

```diff
@@ -33,24 +33,25 @@
 import storm.trident.tuple.TridentTupleView.ProjectionFactory;
 
 public class MapCombinerAggStateUpdater implements StateUpdater<MapState> {
+    //ANY CHANGE TO THIS CODE MUST BE SERIALIZABLE COMPATIBLE OR THERE WILL BE PROBLEMS
+    private static final long serialVersionUID = 4783054195826968400L;
+
     CombinerAggregator _agg;
     Fields _groupFields;
     Fields _inputFields;
     ProjectionFactory _groupFactory;
     ProjectionFactory _inputFactory;
```

Comment:
Because we are setting the serialVersionUID now.  Can we mark _groupFactory and _inputFactory as transient?  They are set in the prepare, so having them not be transient is unneeded.